lms/djangoapps/onboarding/forms.py

```diff
@@ -375,9 +375,7 @@ def save(self, request, commit=True):
 
             prev_org = user_info_survey.organization
             user_info_survey.organization = organization_to_assign
-
-            if organization_to_assign.users_count() == 0:
-                user_info_survey.is_first_learner = True
+            user_info_survey.is_first_learner = True if organization_to_assign.users_count() == 0 else False
```

Comment:
It can be simplified to 
`user_info_survey.is_first_learner = organization_to_assign.users_count() == 0`